packages/react/src/ReactBlockSpec.tsx

```diff
@@ -7,36 +7,42 @@ import {
   blockStyles,
   camelToDataKebab,
   createTipTapBlock,
+  getBlockFromPos,
   mergeCSSClasses,
   parse,
+  Props,
   PropSchema,
   propsToAttributes,
-  render,
+  SpecificBlock,
+  wrapInBlockStructure,
 } from "@blocknote/core";
 import {
   NodeViewContent,
   NodeViewProps,
   NodeViewWrapper,
   ReactNodeViewRenderer,
 } from "@tiptap/react";
+import { renderToString } from "react-dom/server";
 import { createContext, ElementType, FC, HTMLProps, useContext } from "react";
 
 // extend BlockConfig but use a React render function
 export type ReactBlockConfig<
-  Type extends string,
+  BType extends string,
   PSchema extends PropSchema,
   ContainsInlineContent extends boolean,
-  BSchema extends BlockSchema
+  BSchema extends BlockSchema & {
```

Comment:
Do we need a shorthand for this? I think the meaning of this change is `BlockSchemaWithBlock<...>`?